app/Listeners/SendFirstVoteMessage.php

```diff
@@ -27,6 +27,8 @@ public function __construct(MessageBroker $broker)
      */
     public function handle(UserCastFirstVote $event)
     {
+        $country_code_or_global = ! empty($event->user->country_code) ? $event->user->country_code : 'global';
```

Comment:
I think you're right, `isset` would work in this case since the country code is saved as `null` if the header isn't set. I just wanted to be _extra_ sure that we didn't end up with a case where an empty string ended up being passed and that borked the payload. :wink: